libraries/ESP8266HTTPClient/examples/StreamHttpsClient/StreamHttpsClient.ino

```diff
@@ -0,0 +1,116 @@
+/**
+   StreamHTTPClient.ino
+
+    Created on: 24.05.2015
+
+*/
+
+#include <Arduino.h>
+
+#include <ESP8266WiFi.h>
+#include <ESP8266WiFiMulti.h>
+
+#include <ESP8266HTTPClient.h>
+
+#define USE_SERIAL Serial
+
+ESP8266WiFiMulti WiFiMulti;
+
+void setup() {
+
+  USE_SERIAL.begin(115200);
+  // USE_SERIAL.setDebugOutput(true);
+
+  USE_SERIAL.println();
+  USE_SERIAL.println();
+  USE_SERIAL.println();
+
+  for (uint8_t t = 4; t > 0; t--) {
+    USE_SERIAL.printf("[SETUP] WAIT %d...\n", t);
+    USE_SERIAL.flush();
+    delay(1000);
+  }
+
+  WiFi.mode(WIFI_STA);
+  WiFiMulti.addAP("SSID", "PASSWORD");
+
+}
+
+void loop() {
+  // wait for WiFi connection
+  if ((WiFiMulti.run() == WL_CONNECTED)) {
+
+    HTTPClient http;
+
+    BearSSL::WiFiClientSecure client;
+
+    bool mfln = client.probeMaxFragmentLength("tls.mbed.org", 443, 1024);
+    USE_SERIAL.printf("\nConnecting to https://tls.mbed.org\n");
+    USE_SERIAL.printf("MFLN supported: %s\n", mfln ? "yes" : "no");
+    if (mfln) {
+      client.setBufferSizes(1024, 1024);
+    }
+
+    USE_SERIAL.print("[HTTP] begin...\n");
+
+    // configure server and url
+    const uint8_t fingerprint[20] = {0xEB, 0xD9, 0xDF, 0x37, 0xC2, 0xCC, 0x84, 0x89, 0x00, 0xA0, 0x58, 0x52, 0x24, 0x04, 0xE4, 0x37, 0x3E, 0x2B, 0xF1, 0x41};
+    client.setFingerprint(fingerprint);
+
+    //if (http.begin("jigsaw.w3.org", 443, "/HTTP/connection.html", true)) {
+    if (http.begin((Client&) client, "https://tls.mbed.org/")) {
+
+      USE_SERIAL.print("[HTTP] GET...\n");
+      // start connection and send HTTP header
+      int httpCode = http.GET();
+      if (httpCode > 0) {
+        // HTTP header has been send and Server response header has been handled
+        USE_SERIAL.printf("[HTTP] GET... code: %d\n", httpCode);
+
+        // file found at server
+        if (httpCode == HTTP_CODE_OK) {
+
+          // get lenght of document (is -1 when Server sends no Content-Length header)
+          int len = http.getSize();
+
+          // create buffer for read
+          uint8_t buff[128] = { 0 };
+
+          // get tcp stream
+          WiFiClient * stream = &client;
```

Comment:
Same as above, no more "get tcp stream" with your change.

Reply:
getStreamPtr() has been removed from the example